yesod-test/Yesod/Test.hs

```diff
@@ -523,23 +524,26 @@ addFile name path mimetype = do
           addPostData (MultipleItemsPostData posts) contents =
             MultipleItemsPostData $ ReqFilePart name path contents mimetype : posts
 
+-- |
 -- This looks up the name of a field based on the contents of the label pointing to it.
-nameFromLabel :: T.Text -> RequestBuilder site T.Text
-nameFromLabel label = do
+--
+-- @since 1.5.9
```

Comment:
You don't need to give `@since` annotations for non-exported functions

Reply:
No. I understand the rule. I'll do so:)